skeleton-model-app/market/dataInteraction/index.js

```diff
@@ -356,10 +362,82 @@ const handleOrder = async (db, id, payload) => {
   const data = {
     orderId: id,
     ...payload,
+    status: BNPL_ORDER_STATUS.ORDER_STATUS_PENDING,
+    commitments: {}
   }
   await db.Order.create(data)
 }
 
+/**
+ * This function retrieves order commitment values from DB
+ *
+ * @param db
+ * @param orderId
+ * @returns {Promise<Object|null>}
+ */
+const fetchOrderCommitment = async (db, orderId) => {
```

Comment:
@ngo275 I don't see `saveOrderCommitment` method anywhere in the source code so how can we get the order's commitment?

Reply:
I added a column in order table.